eng/internal/NuGet.Template.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
It is unfortunate that these URLs are now duplicated between `$(RestoreSources)` in the root Directory.Build.props file, and here. They will need to stay in sync.

Reply:
Why do we need all these? Don't we only need the feeds that include optional tools and nuget?

Reply:
I just copied all the feeds (that made more or less sense - there was an old xunit feed in it) that were written into the generated template.